src/utils/_mixins.scss

```diff
@@ -0,0 +1,117 @@
+@import "./variables";
```

Comment:
these scss files should be inside styles folder, there (src/utils) i'd expect to see js functions utils